plugins/BEdita/Core/tests/TestCase/Model/Table/ObjectTypesTableTest.php

```diff
@@ -343,15 +354,15 @@ public function testInvalidateCacheAfterDelete()
     {
         $entity = $this->ObjectTypes->get('document');
 
-        $this->assertNotFalse(Cache::read('id_1', ObjectTypesTable::CACHE_CONFIG));
-        $this->assertNotFalse(Cache::read('map', ObjectTypesTable::CACHE_CONFIG));
-        $this->assertNotFalse(Cache::read('map_singular', ObjectTypesTable::CACHE_CONFIG));
+        static::assertNotFalse(Cache::read('id_1', ObjectTypesTable::CACHE_CONFIG));
+        static::assertNotFalse(Cache::read('map', ObjectTypesTable::CACHE_CONFIG));
+        static::assertNotFalse(Cache::read('map_singular', ObjectTypesTable::CACHE_CONFIG));
 
         $this->ObjectTypes->delete($entity);
 
-        $this->assertFalse(Cache::read('id_1', ObjectTypesTable::CACHE_CONFIG));
-        $this->assertFalse(Cache::read('map', ObjectTypesTable::CACHE_CONFIG));
-        $this->assertFalse(Cache::read('map_singular', ObjectTypesTable::CACHE_CONFIG));
+        static::assertFalse(Cache::read('id_1', ObjectTypesTable::CACHE_CONFIG));
```

Comment:
When an object type is removed or its name modified should we invalidate also the relations cache of other object types that can be related to this one?
Since this is not a frequent operation we could simply remove all `id_*` caches